samples/matlab/lithium_ion_battery.m

```diff
@@ -32,18 +26,16 @@
 % - externally-applied current I_app [A]
 % - electrolyte resistance R_elyt [Ohm]
 
-X_Li_an = [0.005:0.025:0.995];
-X_Li_ca = 1 - X_Li_an;
 
-I_app = 0;
-R_elyt = 0;
-T = 300;
-P = oneatm;
-
-global F
-% Parameters
+% Input parameters
+SOC = 0:0.02:1; % [-] Input state of charge (0...1)
+X_Li_an = (0.75-0.01)*SOC+0.01; % anode balancing
+X_Li_ca = (0.99-0.49)*(1-SOC)+0.49; % cathode balancing
```

Comment:
Tiniest little quibble here, @wbessler - might it be clearer for the user if this balancing were made more explicit?

- Would it make sense to specify the upper and lower SOC limits as variables?  I'm not sure if a user would ever want to change them, so there's a risk there, but then the user would understand what 0.01, 0.75, et al. represent
- Or would it be better to just add a comment, explaining what we mean by balancing?
- Lastly, the comment you have removed above, which commented on the anode and cathode capacities being perfectly balanced, has some relevant info here, as well.  Just thinking of ways to make the example a little more transparent to somebody not well versed in battery modeling.